hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/TopicEndpoint.java

```diff
@@ -63,6 +52,16 @@
     TopicMetrics getMetrics(@PathParam("topicName") String qualifiedTopicName);
 
     @POST
+    @Produces(APPLICATION_JSON)
+    @Path("/{topicName}/blacklist")
```

Comment:
just `BlacklistEndpoint`, no need to be so attached to whole `TopicBlacklist` concept, just make it a `Blacklist`, in case we need to ban subscribers :P